app/controllers/schedule_controller.rb

```diff
@@ -74,6 +74,26 @@ def scheduled
     @edition = document.current_edition
   end
 
+  def unschedule
```

Comment:
I think this might benefit from being in its own controller, similar to how we have Withdraw vs. Unwithdraw controllers, as the the actions here are quite separate from each other.

Reply:
Yep i agree with you now i've thought about it, thanks!